packages/desktop/lib/electron/preloads/transak.preload.ts

```diff
@@ -20,6 +21,10 @@ function observeStyleChanges(targetNode: Element): { disconnect: () => void } {
     const observer = new MutationObserver(callback)
     observer.observe(targetNode, config)
 
+    const fallbackTimeout = setTimeout(() => {
+        observer.disconnect()
+    }, 5000)
+
     return {
         disconnect: () => observer.disconnect(),
     }
```

Comment:
Why are we returning this function?

Reply:
So the caller could also control the disconnection, but as we have a fallback it is not needed anymore